package.json

```diff
@@ -44,36 +44,12 @@
       "path": "./node_modules/cz-conventional-changelog"
     }
   },
-  "commitlint": {
-    "extends": [
-      "@commitlint/config-conventional",
-      "@commitlint/config-lerna-scopes"
-    ],
-    "rules": {
-      "body-max-length": [
-        0
-      ],
-      "body-max-line-length": [
-        0
-      ],
-      "footer-max-length": [
-        0
-      ],
-      "footer-max-line-length": [
-        0
-      ],
-      "header-max-length": [
-        0
-      ]
-    }
-  },
   "engines": {
     "node": "^10.12.0 || >=12.0.0"
   },
   "devDependencies": {
     "@commitlint/cli": "^12.0.1",
     "@commitlint/config-conventional": "^12.0.1",
-    "@commitlint/config-lerna-scopes": "^12.0.1",
```

Comment:
Just interested, what's wrong with `@commitlint/config-lerna-scopes` package?

Reply:
current version is now working with lerna 4, PR with fix has been merged but not yet released: https://github.com/conventional-changelog/commitlint/pull/2447